src/igvfd/searches/configs/file.py

```diff
@@ -8,122 +8,287 @@ def file():
             },
             'file_size': {
                 'title': 'File Size',
+                'description': 'File size specified in bytes.',
+                'optional': True,
+                'category': 'File Details',
                 'type': 'stats'
             },
             'sequencing_platform.term_name': {
-                'title': 'Sequencing Platform'
+                'title': 'Sequencing Platform',
+                'description': 'The measurement device used to produce sequencing data.',
+                'category': 'File Details'
+            },
+            'externally_hosted': {
+                'title': 'Externally Hosted',
+                'description': 'Indicates whether the file is externally hosted and not stored on portal.',
+                'optional': True,
+                'category': 'File Details'
+            },
+            'file_format_specifications.standardized_file_format': {
+                'title': 'Standardized Format',
+                'description': 'Boolean value, specifying whether this format is an IGVF-standardized file format defined by Focus Groups or produced by a uniform pipeline.',
+                'optional': True,
+                'category': 'File Details'
+            },
+            'workflows.name': {
+                'title': 'Workflows',
+                'description': 'The workflows used to produce this file.',
+                'optional': True,
+                'category': 'Processing Details'
+            },
+            'workflows.uniform_pipeline': {
+                'title': 'Uniformly Processed',
+                'description': 'The status of the single cell or Perturb-seq uniform pipeline processing for this file.',
+                'optional': True,
+                'category': 'Processing Details'
             },
             'file_set.file_set_type': {
-                'title': 'File Set Type'
+                'title': 'File Set Type',
+                'description': 'The type of file set the file belongs to.',
+                'optional': True,
+                'category': 'File Set Details'
             },
             'assay_titles': {
-                'title': 'Assay Term Names'
+                'title': 'Assay',
+                'description': 'High-level assay classification based on OBI assay slim terms.',
+                'optional': True,
+                'category': 'File Set Details'
             },
             'preferred_assay_titles': {
-                'title': 'Preferred Assay Titles'
+                'title': 'Preferred Assay Titles',
+                'description': 'Title of assays that generated the measurement set.',
+                'category': 'File Set Details'
             },
             'cell_type_annotation.term_name': {
-                'title': 'Annotated Cell Type'
+                'title': 'Annotated Cell Type',
+                'description': 'The inferred cell type this file is associated with based on single-cell expression profiling.',
```

Comment:
"The inferred cell type cluster from single-cell profiling that this file represents"